drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
 mdb = mdb.withName("metricName").withType(MetricType.COUNTER)
   .reusable(resolveIsReusable());
 ----
+[#JMO]
+== Jaeger for MicroProfile OpenTracing
```

Comment:
Make this title match the link above (with the brackets and feature names on the end).